### ISSUE TITLE
[CI] [Buildkite] Feature request: automatically upload core dumps from FreeBSD jobs

### ISSUE THREAD

**Issue by DilumAluthge** [JuliaLang/julia#42793]

**Comment by ararslan** [comment]
Note: Needs https://github.com/JuliaLang/julia/pull/42990 as a prerequisite

**Comment by ararslan** [comment]
Not really, as FreeBSD CI doesn't yet run on buildkite and some of the scripts in that repository won't work as-is on FreeBSD (in particular, they use `#!/bin/bash`, which is not a thing on FreeBSD).
